docs/content/jottacloud.md

```diff
@@ -233,16 +233,17 @@ them. Generally you should avoid these, unless you know what you are doing.
 
 ### --fast-list
 
-This remote supports `--fast-list` which allows you to use fewer
+This backend supports `--fast-list` which allows you to use fewer
 transactions in exchange for more memory. See the [rclone
 docs](/docs/#fast-list) for more details.
 
 Note that the implementation in Jottacloud always uses only a single
 API request to get the entire list, so for large folders this could
 lead to long wait time before the first results are shown.
 
-Note also that with rclone version 1.58 and newer information about
-[MIME types](/overview/#mime-type) are not available when using `--fast-list`.
+Note also that with rclone version 1.58 and newer, information about
+[MIME types](/overview/#mime-type) and metadata item [utime](#metadata)
+are not available when using `--fast-list`.
```

Comment:
So if you need metadata you can't use `ListR` - is that correct? That is quite a limitation. We should probably say here use `--disable ListR` to work around or something like that?

Reply:
> So if you need metadata you can't use ListR - is that correct?

Not quite. You'll get the `btime` (created) and `mtime` (modified) timestamps, just not `utime` (updated) which I don't see as that big of a limitation. It does not have `content-type` (MIME) either, but that's already a documented limitation with ListR so it just extends to the metadata section.